src/main/java/com/github/manolo8/darkbot/Bot.java

```diff
@@ -55,7 +55,11 @@ public static void main(String[] args) throws IOException {
             UIManager.put("Table.cellFocusColor", new Color(0, 0, 0, 160));
 
             // set icon here to use it in login popup, java check popup etc.
-            Popups.setDefaultIcon(MainGui.ICON);
+            try {
+                Popups.setDefaultIcon(MainGui.ICON);
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
```

Comment:
i wonder, what's the motive behind this change? is it throwing an exception under some systems? if so, which exception?